source/sync/data-model/update-schema.txt

```diff
@@ -1,441 +1,589 @@
 
-
-When developing an application using Atlas Device Sync, you may need to make changes to
-your schema at some point, such as when you need to: 
-
-- Add a new property to an already-synced object
-- Remove a property from an already-synced object
-- Change the type stored in a property
-- Update an optional property to *required* 
-
-To make it easier to understand how schema changes affect your app, we
-characterize them as "breaking" versus "non-breaking" changes.
-
-Atlas App Services provides for non-breaking schema changes to synced 
-realms, allowing old clients to sync with newer ones.
-
-Breaking schema changes, however, take some planning and work, and should
-be avoided whenever possible.
+.. toctree::
+   :titlesonly:
+   :hidden:
+
+   Update Data Model in Development </sync/data-model/update-data-model/update-data-model-development>
+   Update Data Model in Production </sync/data-model/update-data-model/update-data-model-production>
+   Migrate Data Model to Partner Collection </sync/data-model/update-data-model/migrate-schema-partner-collection>
+
+You will most likely need to make changes to your app's
+:ref:`data model <device-sync-data-model>` after you have defined it. For
+example, while developing your app, you may realize that an existing
+property should be required. Or while planning for a future release for your
+production-level app, you determine that a feature requires a new object type.
+
+In general, we refer to data model updates in the following terms:
+
+What type of change are you making?
+  We broadly categorize changes to a data model as being either *breaking* or
+  *non-breaking*, depending on whether they break compatibility with your
+  current model. These change types are discussed in more detail below, or you
+  can check out the :ref:`breaking-change-quick-reference` section.
+
+Where are you making the change?
+  Because Device Sync requires a matching client-side SDK data model and
+  server-side App Services schema, we discuss data model changes in terms of
+  whether you update the *client side* or the *server side* first.
+  Depending on the change you make, there are implications for which side
+  you should update first and how you apply those same changes to the other
+  side. This is discussed in more detail below.
+
+Is your app in development or production?
+  The recommended process for updating your data model also depends on whether
+  you are updating an app in production or in development:
+
+  - If your app is in development, you can use Device Sync's
+    :ref:`Development Mode <development-mode>` feature to streamline data model
+    changes, including breaking changes. To learn more, refer to
+    :ref:`update-data-model-development-mode`.
+
+  - If you are updating a production-level app, data model changes require
+    additional planning to reduce the impact to existing clients. To learn more,
+    refer to :ref:`update-data-model-production`.
+
+Change Types
```

Comment:
[nit] could rename to "Types of Changes". IMO change types seems like we're talking changing the types of properties